clang/include/clang/Driver/Driver.h

```diff
@@ -582,6 +582,10 @@ class Driver {
   /// @name Helper Methods
   /// @{
 
+  /// MakeSYCLDeviceTriple - Returns the SYCL device triple for the
+  /// specified subarch
+  llvm::Triple MakeSYCLDeviceTriple(StringRef TargetArch = "spir64") const;
```

Comment:
Will it be possible to use spirv64 as default here? Thanks

Reply:
Using `spirv64` wasn't discussed as a default.  @bader do you have any thoughts on this?

Reply:
I think using spirv64 by default in the long term makes sense. `spir64` is associated with the LLVM bitcode format following the SPIR-1.2 spec (i.e. using spir/spi64 target triple, certain metadata, etc.) adjusted to recent LLVM IR version.

@asudarsa, do you imply that we should use SPIR-V back-end to produce the SPIR-V instead of spir-targeted LLVM IR?

Reply:
Hi @bader 

Yes. That's the eventual goal. However, we are not yet ready to transition from the SPIR-V Translator to the Backend. I think we should be able to use spirv64 with the translator as well. 

Thanks

Reply:
Offline quick convo with @asudarsa concluded we will stick with spir64 initially.